app.py

```diff
@@ -468,7 +471,8 @@ def jurisdictions_set(election_id):
             audit_board_obj = AuditBoard(
                 id = audit_board["id"],
                 name = audit_board["name"],
-                jurisdiction_id = jurisdiction_obj.id
+                jurisdiction_id = jurisdiction_obj.id,
+                passphrase = xp.generate_xkcdpassword(WORDS, numwords=4, delimiter="-")
```

Comment:
Just to clarify, we don't care that these will be different given the same seed, right?

Reply:
correct, the consistency we care about is for the audit data itself, not for authentication.